wandb/sdk/internal/sender.py

```diff
@@ -1038,6 +1084,8 @@ def _init_run(
             self._run.resumed = True
             if self._resume_state.wandb_runtime is not None:
                 self._run.runtime = self._resume_state.wandb_runtime
+        elif is_rewinding:
+            self._setup_rewind()
```

Comment:
it's not clear from the code why `_setup_rewind()` is not always called when `is_rewinding` is true

Reply:
added comment